hazelcast-client-new/src/main/java/com/hazelcast/client/cache/impl/ClientCacheProxy.java

```diff
@@ -235,12 +235,9 @@ public void clear() {
         }
         final Data keyData = toData(key);
         Data epData = toData(entryProcessor);
-        List<Data> argumentsData = null;
-        if (arguments != null) {
-            argumentsData = new ArrayList<Data>(arguments.length);
-            for (int i = 0; i < arguments.length; i++) {
-                argumentsData.add(toData(arguments[i]));
-            }
+        List<Data> argumentsData = new ArrayList<Data>(arguments.length);
```

Comment:
It seems if user passes null we get NULL arguments. I was expecting to get a list with 1 element whose first element is null. Fixing it now.